code/game/gamemodes/gang/gang_things.dm

```diff
@@ -9,7 +9,7 @@
 /obj/item/gang_induction_package/attack_self(mob/living/user)
 	..()
 	if(HAS_TRAIT(user, TRAIT_MINDSHIELD))
-		to_chat(user, "You attended a seminar on not signing up for a gang, and are not interested.")
+		to_chat(user, "You are mindshielded, so you cannot join a family.")
```

Comment:
These are dubbed loyalty implants, you can build on that here instead of being all game mechanics.

ie something along the lines of "This kinda thing wasn't mentioned in the company approved extracurricular activities brochure." or whatever.

Reply:
???

Reply:
they used to be called loyalty, we changed it so that people stopped assuming they had some sort of additional roleplay requirement.

I think it would be better if it was less blatant on the mindshield mechanic protecting you which is what I think skog was trying to get at.

```suggestion
		to_chat(user, "Your mind vehemently refuses to accept the family's ideologies.")
```